src/System.Text.Json/tests/Serialization/DictionaryTests.KeyPolicy.cs

```diff
@@ -308,5 +311,94 @@ private class ClassWithExtensionDataProperty
            [JsonExtensionData]
             public Dictionary<string, object> Data { get; set; }
         }
+        
+        [Fact]
+        public static void CamelCaseSerialize_ForTypedDictionary_ApplyDictionaryKeyPolicy()
+        {
+            const string JsonCamel = @"{""keyDict"":{""Name"":""text"",""Number"":1000,""isValid"":true,""Values"":[1,2,3]}}";
+            var obj = new Dictionary<string, CustomClass>()
```

Comment:
Nit: please add new lines between these statements here and below for readability.